EntityFramework.Utilities/EntityFramework.Utilities/SqlQueryProvider.cs

```diff
@@ -15,7 +15,7 @@ public class SqlQueryProvider : IQueryProvider, INoOpAnalyzer
 		public bool CanBulkUpdate => true;
 
 		private static readonly Regex FromRegex = new Regex(@"FROM \[([^\]]+)\]\.\[([^\]]+)\] AS (\[[^\]]+\])", RegexOptions.IgnoreCase | RegexOptions.Compiled);
-		private static readonly Regex UpdateRegex = new Regex(@"(\[[^\]]+\])[^=]+=(.+)", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+		private static readonly Regex UpdateRegex = new Regex(@"(\[[^\]]+\])[^=]+(=|IS)(.+)", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
I would prefer you use a [non-capturing group](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Non-capturing_group) if you're not intending to read the value from this group.

So the regex should be: `(\[[^\]]+\])[^=]+(?:=|IS)(.+)`. And further down it should be reverted to `match.Groups[2]`.

Reply:
Oh neat, didn't even know that was a thing!